[W-10728932] add draft content for new home page (#114)
mulesoft/docs-general: modules/ROOT/pages/index.adoc

= [.brand]#MuleSoft# Documentation
:page-layout: home
:page-fragmentize:
:!sectids:
ifndef::env-site[:imagesdir: ../images]

[#cta]
xref:api-led-overview.adoc[New to MuleSoft? Start here]

[#the-road]
== The Road to Digital Transformation

image::getting-started.png[]

--
[discrete]
=== Getting Started

Build the digital transformation your business needs.

* xref:api-led-overview.adoc[Tutorial: Build and Deploy an API]
--

image::discover.png[]

--
[discrete]
=== Discover

Browse Exchange to find existing API specifications, templates, examples, and other assets that you can reuse for your projects.

* https://www.anypoint.mulesoft.com/exchange/[Discover reusable assets]
--

image::design.png[]

--
[discrete]
=== Design

Create and publish new API specifications from scratch.

> * xref:#[Create a reusable API specification]
> * xref:#[Plan for access]
> * xref:#[Plan for security]

Reviewer: ```suggestion
* xref:design-center::design-create-publish-api-specs.adoc[Create a reusable API specification]
* xref:access-management::index.adoc[Plan for access]
* xref:general::security.adoc[Plan for security]
```

--

image::develop-and-test.png[]

--
[discrete]
=== Develop & Test

Build and test APIs and integrations.

> * xref:#[Develop your API]
> * xref:#[Connect your data, systems, and apps]
> * xref:#[Develop the Mule app that integrates systems, services, APIs, and devices]
> * xref:#[Build automated tests for your APIs and integrations]

Reviewer: ```suggestion
* xref:studio::index.adoc[Develop your API]
* xref:connectors::index.adoc[Connect your data, systems, and apps]
* xref:mule-runtime::mule-app-dev.adoc[Develop the Mule app that integrates systems, services, APIs, and devices]
* xref:dataweave::index.adoc[Access and transform data within the Mule app]
* xref:munit::index.adoc[Build automated tests for your APIs and integrations]
```

--

image::deploy.png[]

--
[discrete]
=== Deploy

Deploy APIs and apps to your hosting targets, and secure them with policies.

> * xref:#[Deployment options]
> * xref:#[Configure a proxy for your API]
> * xref:#[Create policies for your API]

Reviewer: ```suggestion
* xref:runtime-manager::deployment-strategies.adoc[Deployment options]
* xref:api-manager::getting-started-proxy.adoc[Configure a proxy for your API]
* xref:api-manager::policies-landing-page.adoc[Create policies to secure your API]
```

--

image::monitor.png[]

--
[discrete]
=== Monitor

Monitor your APIs and integrations using dashboards, metrics, and visualization.

> * xref:#[Monitor Mule app performance]
> * xref:#[Monitor deployed API performance]
> * xref:#[Monitor deployed Mule app performance]

Reviewer: ```suggestion
* xref:monitoring::index.adoc[Monitor Anypoint Platform performance from a single user interface]
* xref:api-functional-monitoring::index.adoc[Monitor your deployed API performance]
* xref:runtime-manager::monitoring.adoc[Monitor your deployed apps]
```

--

== Quick Links

[#popular-topics]
=== Popular Topics

* xref:#[CloudHub]
* xref:#[Connectors]
* xref:#[DataWeave Language]
* xref:#[Design Center]
* xref:#[Exchange]

> * xref:#[Mule Runtime 4.3]

Reviewer 2: Not sure if this is a placeholder, but I suspect we don't want to point people to 4.3 when 4.4 is the latest even if 4.3 is currently more popular at the moment. A company goal is to get people using the latest, and we're moving away from versioning in the future. 

Same very likely goes for the Studio versions.

Author: This text came from the Figma design, so I think it's safe to say it was a placeholder. To solve it, I would recommend dropping the mention of the version. Once set up, the URL will resolve to the latest version.

> * xref:#[CloudHub]
> * xref:#[Connectors]
> * xref:#[DataWeave Language]
> * xref:#[Design Center]
> * xref:#[Exchange]
> * xref:#[Mule Runtime 4.3]
> * xref:#[Studio 7.4]

Reviewer: ```suggestion
* xref:dataweave::dw-core-functions-filter.adoc[Dataweave Function: filter]
* xref:dataweave::dataweave-cookbook-format-dates.adoc[Format Dates and Times]
* xref:dataweave::dw-operators.adoc[DataWeave Operators]
* xref:connectors::index.adoc[Connectors]
* xref:mule-runtime::mule-error-concept[Mule Errors]
* xref:mule-runtime::secure-configuration-properties.adoc[Secure Configuration Properties]
* xref:dataweave::dw-core-functions-map.adoc[Dataweave Function: map]
```

[#latest-releases]
=== Latest Releases

[cols=10;90]
|===

> |June 4
> |xref:#[Anypoint Security Edge]
> 
> |June 2
> |xref:#[Mule runtime engine for CloudHub]
> 
> |June 1
> |xref:#[API Designer 2.17.2]
> 
> |June 1
> |xref:#[MUnit Studio Plugin 2.5.1a]
> 
> |May 30
> |xref:#[API Designer 2.17.0a]

Reviewer: ```suggestion
|April 11
|xref:api-functional-monitoring/api-functional-monitoring-release-notes.adoc[API Functional Monitoring 2.1]

|April 11
|xref:connector/connector-db.adoc#1-13-0[Database Connector 1.13]

|April 2
|xref:design-center/design-center-release-notes-api_specs.adoc#2-41-0[API Designer (Crowd) Release Notes 2.41]

|April 2
|xref:api-mocking-service/api-mocking-service-release-notes.adoc#2-38-0[API Mocking Service Release Notes 2.38]

|March 31
|xref:studio/anypoint-studio-7.12-with-4.4-runtime-release-notes.adoc[Anypoint Studio 7.12]
```

|===

xref:#[View all release notes,role=view-all]

[#more-resources]
=== More Resources

> * xref:#[MuleSoft Help Center]

Reviewer: ```suggestion
* https://help.mulesoft.com/s/[MuleSoft Help Center^]
```

> * xref:#[Developer Resources]

Reviewer: ```suggestion
* https://developer.mulesoft.com/tutorials-and-howtos/getting-started/hello-mule[Developer Tutorials^]
```

> * xref:#[MuleSoft Training]

Reviewer: ```suggestion
* https://training.mulesoft.com/[Training^]
```

> * xref:#[MuleSoft Basics on Trailhead]

Reviewer: ```suggestion
* https://trailhead.salesforce.com/content/learn/modules/mulesoft-basics[Learn the Basics of MuleSoft^]
```

* xref:munit::index.adoc[Build automated tests for your APIs and integrations]
--

image::deploy.png[]

--
[discrete]
=== Deploy

Deploy APIs and apps to your hosting targets, and secure them with policies.

* xref:runtime-manager::deployment-strategies.adoc[Deployment options]
* xref:api-manager::getting-started-proxy.adoc[Configure a proxy for your API]
* xref:api-manager::policies-landing-page.adoc[Create policies to secure your API]
--

image::monitor.png[]

--
[discrete]
=== Monitor

Monitor your APIs and integrations using dashboards, metrics, and visualization.

* xref:monitoring::index.adoc[Monitor Anypoint Platform performance from a single user interface]
* xref:api-functional-monitoring::index.adoc[Monitor your deployed API performance]
* xref:runtime-manager::monitoring.adoc[Monitor your deployed apps]
--

== Quick Links

[#popular-topics]
=== Popular Topics

* xref:dataweave::dw-core-functions-filter.adoc[Dataweave Function: filter]
* xref:dataweave::dataweave-cookbook-format-dates.adoc[Format Dates and Times]
* xref:dataweave::dw-operators.adoc[DataWeave Operators]
* xref:connectors::index.adoc[Connectors]

> * xref:mule-runtime::mule-error-concept[Mule Errors]

Reviewer 2: ```suggestion
* xref:mule-runtime::mule-error-concept.adoc[Mule Errors]
```

* xref:mule-runtime::secure-configuration-properties.adoc[Secure Configuration Properties]
* xref:dataweave::dw-core-functions-map.adoc[Dataweave Function: map]

[#latest-releases]
=== Latest Releases

[cols=10;90]
|===
|April 11
|xref:api-functional-monitoring/api-functional-monitoring-release-notes.adoc[API Functional Monitoring 2.1]

|April 11
|xref:connector/connector-db.adoc#1-13-0[Database Connector 1.13]

|April 2
|xref:design-center/design-center-release-notes-api_specs.adoc#2-41-0[API Designer (Crowd) Release Notes 2.41]

|April 2
|xref:api-mocking-service/api-mocking-service-release-notes.adoc#2-38-0[API Mocking Service Release Notes 2.38]

|March 31
|xref:studio/anypoint-studio-7.12-with-4.4-runtime-release-notes.adoc[Anypoint Studio 7.12]
|===

> xref:#[View all release notes,role=view-all]

Reviewer: ```suggestion
xref:release-notes::quick-refs/by-date-index.adoc[Release Notes by Month,role=view-all]
```
